### PR TITLE
Issue #19: Fetch logged user activities

### PR DESCRIPTION
**Pull request by islam01jul00**
Now we have more 4 endpoints

GET /api/user/activities => to retrieve authenticated user activities
GET /api/user/activities/:activityId => to retrieve authenticated user activity by id

GET /api/activities => to retrieve all activities
GET /api/activities/:activityId => to retrieve activity by id

### CODE REVIEW

**Review by zekooo0 — APPROVED** [review]
LGTM

File: Server/Routes/activitiesRoutes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const verifyRole = require('../utils/verifyRole');
+const { getActivities, getActivity } = require('../Controllers/activitiesController');
+
+const router = require("express").Router();
+
+// I don't think this resourse can be public so i limited it for admin.
+
+router.route('/').get(verifyRole('admin'), getActivities);
+router.route('/:activityId').get(verifyRole('admin'), getActivity);
```
Comment (AhmedNagehAbdelrazek) [inline Server/Routes/activitiesRoutes.js:9]: the same here too

File: Server/Controllers/userActivitiesController.js
Comment (AhmedNagehAbdelrazek) [inline Server/Controllers/userActivitiesController.js:1]: you don't need to create another controller to control the activities, combine both in `activitiesController.js`

**Review by AhmedNagehAbdelrazek — CHANGES_REQUESTED** [review]
the functionality is working fine 
but it would be best if you fixed the other issues

**Review by AhmedNagehAbdelrazek — APPROVED** [review]

**Pull request merged**